tests/integration/standard/test_ip_change.py

```diff
@@ -16,6 +16,7 @@ def setup_module():
     use_cluster('test_ip_change', [3], start=True)
 
 @local
+@scylla_only
```

Comment:
Can we have some string or comment that will describe reason why it is only supported by scylla ?

Reply:
seems like test is setting `api_address` which isn't legal for cassandra
I'll try fixing the test

Reply:
just removing it, works fine of both scylla and cassnadra